ecocodejava.properties

```diff
@@ -2,25 +2,25 @@ category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/ecoCode
 publicVersions=1.4.2
-archivedVersions=1.3.0
+archivedVersions=1.2.1,1.3.0
 
 defaults.mavenGroupId=io.ecocode
 defaults.mavenArtifactId=ecocode-java-plugin
 
 1.2.1.description=Initial Marketplace publication of plugin for SonarQube 9.9.+ LTS
-1.2.1.sqVersions=[9.9,10.1]
+1.2.1.sqVersions=[9.9,10.3.*]
 1.2.1.date=2023-04-14
 1.2.1.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.2.1/ecocode-java-plugin-1.2.1.jar
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Update rules tags�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,10.1]
+1.3.0.sqVersions=[9.9,10.3.*]
 1.3.0.date=2023-07-04
 1.3.0.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.3.0/ecocode-java-plugin-1.3.0.jar
 1.3.0.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.3.0
 
-1.4.2.description=Move JavaScript, PHP and Python plugins to their dedicated repositories ; [TEC] Compatibility with SonarQube 10.1 ; Creation and publishing ecocode-rules-specifications module (new rules reference module) ; Improve implementation of rules EC1, EC2, EC34, EC35 ; Upgrade rules EC3, EC53, EC67 specifications ; Depreciation of EC4 rule for Java ; Adding iOS and Android rules to ecocode-rules-specifications
-1.4.2.sqVersions=[9.9,10.1]
+1.4.2.description=Move JavaScript, PHP and Python plugins to their dedicated repositories ; Compatibility with SonarQube 10.1 ; Creation and publishing ecocode-rules-specifications module (new rules reference module) ; Improve implementation of rules EC1, EC2, EC34, EC35 ; Upgrade rules EC3, EC53, EC67 specifications ; Depreciation of EC4 rule for Java ; Adding iOS and Android rules to ecocode-rules-specifications
+1.4.2.sqVersions=[9.9,10.3.*]
```

Comment:
By not ending this range with `LATEST`, you've ensured that this plugin won't be offered in the Marketplace for 10.4 when it's released. Is that the intent?

Reply:
done